treeherder/webapp/api/performance_serializers.py

```diff
@@ -653,6 +654,200 @@ class Meta:
         ]
 
 
+class StandardStatsSerializer(serializers.Serializer):
+    min = PerfCompareDecimalField()
+    max = PerfCompareDecimalField()
+    mean = PerfCompareDecimalField()
+    median = PerfCompareDecimalField()
+    stddev = PerfCompareDecimalField()
+    count = serializers.IntegerField()
+    variance = PerfCompareDecimalField()
+    stddev_pct = PerfCompareDecimalField()
+
+
+class StatisticsTestSerializer(serializers.Serializer):
+    test_name = serializers.CharField()
+    stat = PerfCompareDecimalField()
+    pvalue = PerfCompareDecimalField(required=False)
+    interpretation = serializers.CharField(required=False)
+
+
+class SilvermanKDESerializer(serializers.Serializer):
+    bandwidth = serializers.CharField()
+    base_mode_count = serializers.IntegerField()
+    new_mode_count = serializers.IntegerField()
+    mode_comments = serializers.ListField(child=serializers.CharField(), default=[])
+    warnings = serializers.ListField(child=serializers.CharField(), default=[])
+    mode_summary = serializers.CharField()
+    median_shift_summary = serializers.CharField()
+    ci_low = PerfCompareDecimalField()
+    ci_high = PerfCompareDecimalField()
+    shift = PerfCompareDecimalField()
+    shift_summary = serializers.CharField()
+    is_regression = serializers.BooleanField()
+    is_improvement = serializers.BooleanField()
+    ci_warning = serializers.CharField(required=False)
+
+
+class KDESerializer(serializers.Serializer):
+    median = serializers.CharField()
+    sample_count = serializers.IntegerField()
+    kde_x = serializers.ListField(child=PerfCompareDecimalField(), default=[])
+    kde_y = serializers.ListField(child=PerfCompareDecimalField(), default=[])
+
+
+class CLESSerializer(serializers.Serializer):
+    cles = PerfCompareDecimalField()
+    cles_direction = serializers.CharField(default="")
+    mann_whitney_u_cles = serializers.CharField(default="")
+    p_value_cles = serializers.CharField(default="")
+    cliffs_delta_cles = serializers.CharField(default="")
+    effect_size = serializers.CharField(default="")
+    cles_explanation = serializers.CharField(default="")
+
+
+class PerfCompareResultsSerializerV2(serializers.ModelSerializer):
+    base_rev = serializers.CharField()
+    new_rev = serializers.CharField()
+    base_rev_data = serializers.ListField(child=PerfCompareDecimalField(), default=[])
```

Comment:
`base_rev_data` and `base_runs` contain the same data, I would remove `base_rev_data` and keep `base_runs` to keep the consistency between PerfCompareResultsSerializerV2 and PerfCompareResultsSerializer

Reply:
Same for `new_rev_data` and `new_runs`